Source/WebKit/NetworkProcess/NetworkDataTask.h

```diff
@@ -162,6 +164,7 @@ class NetworkDataTask : public ThreadSafeRefCountedAndCanMakeThreadSafeWeakPtr<N
     void scheduleFailure(FailureType);
 
     void restrictRequestReferrerToOriginIfNeeded(WebCore::ResourceRequest&);
+    void updateNetworkUsage(size_t usage) { m_networkUsage = usage; }
```

Comment:
nit: seems like this should just be named `setNetworkUsage`.

"NetworkUsage" also seems pretty vague to me--maybe "bytesTransferredOverNetwork" or something?